### DISCUSSION TITLE
Toolbar cannot display icon

### DISCUSSION THREAD

**Discussion by xxphjm** [rustdesk/rustdesk#7262] [General]
### Bug Description

1.When the entire restart occurs, the situation shown in Figure 1 will occur.
2.After a while, the connection will be reconnected and the resolution will be reset.
3.When the computer restarts, there is nothing on the reconnect screen.( I can not steps take to reproduce this behavior)
4.no sound from remote side 

### How to Reproduce

As above

### Expected Behavior

1.Show icon
2. Can always be connected
3. can show window
4. There is sound from the remote side

### Operating system(s) on local side and remote side

mac->windows 11

### RustDesk Version(s) on local side and remote side

1.2.3

### Screenshots

![截圖 2024-02-25 22 15 20](https://github.com/rustdesk/rustdesk/assets/71825356/56461ad0-cb65-4249-874b-fe67e3febc06)

**Comment by xlionjuan** [comment]
try nightly or disable hardware codec, and you should provide more details especially your Mac, is it ARM Mac or Intel Mac? It can cause different behavior

And your Bug Description, your expression isn't really clear
1. I could see a screenshot with windows toolbar inside your RustDesk, so I assume this is* from Mac just like you said
2. Which resolution? Windows 11 itself? or your RustDesk isn't render correctly?
3. Couldn't understand, you mean it stuck on circling?
